src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -202,6 +186,30 @@ private async Task<string> GetMetadataSchema(Uri url)
             };
         }
 
+        private async Task<string> FetchRetry(string url, IReadOnlyDictionary<string, string> headers = null, bool nullOn404 = false)
```

Comment:
This better be named `FetchValidationRules` as it writes a validation specific error message on failure, to prevent other people from misusing this for other purposes.

```suggestion
        private async Task<string> FetchValidationRules(string url)
```

Reply:
Done